VKappProject/VKappProject/Controllers/FriendListTableViewController.swift

```diff
@@ -14,60 +15,76 @@ final class FriendListTableViewController: UITableViewController {
 
     // MARK: - Private Properties
 
-    private let users: [User] = [
-        User(
-            avatarImageName: Constants.UserImageNames.bryleeCollinsImageName,
-            name: Constants.UserNames.bryleeCollinsName
-        ),
-        User(avatarImageName: Constants.UserImageNames.emilieRivasImageName, name: Constants.UserNames.emilieRivasName),
-        User(avatarImageName: Constants.UserImageNames.connorLloydImageName, name: Constants.UserNames.connorLloydName),
-        User(avatarImageName: Constants.UserImageNames.ireneNormanImageName, name: Constants.UserNames.ireneNormanName),
-        User(
-            avatarImageName: Constants.UserImageNames.kamronThomasImageName,
-            name: Constants.UserNames.kamronThomasName
-        ),
-        User(
-            avatarImageName: Constants.UserImageNames.kaylynnGrossImageName,
-            name: Constants.UserNames.kaylynnGrossName
-        ),
-        User(avatarImageName: Constants.UserImageNames.kyleRosalesImageName, name: Constants.UserNames.kyleRosalesName),
-        User(avatarImageName: Constants.UserImageNames.martinSteinImageName, name: Constants.UserNames.martinSteinName)
-    ]
-
-    private var sections: [Character: [User]] = [:]
+    private var users: [UserItem] = []
+    private var sections: [Character: [UserItem]] = [:]
     private var sectionTitles: [Character] = []
+    private let service = VKAPIService()
```

Comment:
```suggestion
    private let vkApiService = VKAPIService()
```
Я бы назвал так. Из названия непонятно, что за сервис.

Reply:
Это исправлял после коммента Жени